attributes/network/14_proof.adoc

```diff
@@ -0,0 +1,28 @@
+= Proof Attribute (#14)
```

Comment:
This file should be `response/3_proof.adoc`, since it describes the attribute argument of the network Proof attribute (#14).

You need only to describe the argument here. See async-related example in `response/1_async.adoc`

Reply:
@fmorency OK, I can move this file.